.editorconfig

```diff
@@ -1,8 +1,253 @@
 root = true
 
 [*]
-indent_style = space
-end_of_line = lf
 charset = utf-8
+end_of_line = lf
```

Comment:
FYI - On Windows, files checked out by git start out with CRLF line ending by default, but then due to this setting, get converted to LF (at least by my Windows InteliJ IDE) upon modifying and saving a file. However, this doesn't seem to cause any trouble as Windows InteliJ is probably smart enough to handle LF fine as well. And as we've already had this (end_of_line) in for a while and some of us are Windows users, this is probably safe to keep.

Reply:
@stevesap You may eventually want to configure core.autocrlf for that repository to make git enforce a certain line ending style: https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration (in case you don't trust the IDE or use other editors additionally).